### PR TITLE
feat(ingestion): add aws athena ingestion source

### PR DESCRIPTION
**Pull request by thomasplarsson**
## Checklist
- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

File: metadata-ingestion/README.md [hunk 1]
```diff
@@ -295,6 +295,29 @@ source:
...
+```yml
+source:
+  type: athena
+  config:
+     username: aws_access_key_id # Optional. If not specified, credentials are picked up according to boto3 rules
+     # See https://boto3.amazonaws.com/v1/documentation/api/latest/guide/credentials.html
+     password: aws_secret_access_key # Optional.
+     database: database # Optional, defaults to "default"
+     aws_region: aws_region_name # i.e. "eu-west-1"
+     s3_output_location: s3_location # "s3://<bucket-name>/prefix/"
```
Comment (hsheth2) [inline metadata-ingestion/README.md:316]: I found this a bit confusing at first - would be good to add a quick explanation/link here

**Review by hsheth2 — APPROVED** [review]
Awesome stuff!

**Review by shirshanka — APPROVED** [review]
LGTM!

File: metadata-ingestion/README.md (hunk 1 above)
Comment (thomasplarsson) [reply metadata-ingestion/README.md:316]: Agree. Improvement PR coming up shortly.

**Pull request merged**